database/migrations/2021_12_09_093910_alter_products_details_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AlterProductsDetailsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('products_details', function (Blueprint $table) {
+            $table->renameColumn('fabric', 'composition');
+            $table->tinyText('prod_features')->default('')->after('fabric');
```

Comment:
I recommend to rename `prod_features` to `features` as you already are in the context of the product details.

Reply:
I think it will be better too. Renamed.